src/source/Main.scala

```diff
@@ -202,6 +203,8 @@ object Main {
       opt[String]("objc-h-ext").valueName("<ext>").foreach(objcHeaderExt = _)
         .text("The filename extension for Objective-C[++] header files (default: \"h\")")
       opt[String]("objc-type-prefix").valueName("<pre>").foreach(objcTypePrefix = _)
+        .text("The filename extension for Objective-C[++] header files (default: \"h\")")
+      opt[String]("objc-type-suffix").valueName("<pre>").foreach(objcTypeSuffix = _)
```

Comment:
Should we use `<suffix>` for the valueName here?

Reply:
yeah, or `su`